specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -24,10 +24,12 @@ To see additional help and options, run:
 
 These are the global settings for the Machine Learning Services API.
 
+
 ``` yaml
 openapi-type: arm
-tag: package-2021-04-01
+tag: package-preview-2021-03
```

Comment:
Can you double check that this tag is ok? I think this was the tag generated by OpenAPI Hub. It's different from previous ones, is this ok?

Reply:
Changing the default tag might have impacts to some SDKs that relies on this. Please double confirm we truly need this as well as the impact for the SDKs

@msyyc for python
@qiaozha for JS
@weidongxu-microsoft for Java

Reply:
For Java, it is OK to change as SDK for MLS is in preview. However we'd better know the reason. Preferable SDK would like stable version (e.g. 2021-04-01 here), but the stable version is 4 json less than the preview version...

Reply:
I went ahead and changed this back to `package-2021-04-01` for now. We can handle release request issues separately from this PR